wiki/People/BAT/en.md

```diff
@@ -0,0 +1,84 @@
+# Titanic Beatmap Approval Team
```

Comment:
```suggestion
# Beatmap Approval Team
```

Would maybe fit better

Reply:
oh true, that probably means the other should be changed as well